docs(geo): update existing resource flow (#3468)
aws-amplify/docs: docs/lib/geo/fragments/js/maps.md

## Display a map

> First, make sure you've provisioned a map resource and configured your app using the instructions in either [Getting started](~/lib/geo/getting-started.md) or [Use existing AWS resources](~/lib/geo/existing-resources.md).

Reviewer: ```suggestion
First, make sure you've provisioned a map resource and configured your app using the instructions in either [Getting started](~/lib/geo/getting-started.md) or [Use existing Amazon Location Service resources](~/lib/geo/existing-resources.md).
```

> First, make sure you've provisioned a map resource and configured your app using the instructions in either [Getting started](~/lib/geo/getting-started.md) or [Use existing Amazon Location Service resources](~/lib/geo/existing-resources.md).

Reviewer 2: ```suggestion
First, ensure you've provisioned an Amazon Location Service Map resource and configured your app using the instructions in either [Getting started](~/lib/geo/getting-started.md) or [Use existing resources](~/lib/geo/existing-resources.md) guide.
```

> To render a map, the [MapLibre GL](https://github.com/maplibre/maplibre-gl-js) library and the `maplibre-gl-js-amplify` package are required. MapLibre GL is an open source map rendering library for JavaScript. `maplibre-gl-js-amplify` is a library that makes it easy to integrate MapLibre with Amplify Geo.

Reviewer 2: ```suggestion
To render a map, the [MapLibre GL](https://github.com/maplibre/maplibre-gl-js) and the `maplibre-gl-js-amplify` libraries are required. MapLibre GL is an open source map rendering library and`maplibre-gl-js-amplify` library makes it easy to integrate MapLibre with Amplify Geo and handles Authentication.
```

npm install -S maplibre-gl maplibre-gl-js-amplify
```

Next, configure the Amplify library in your app. All geo-related configuration information is available within the `aws-exports.js` file. Add the following to your app's entry point (i.e. App.js):
Import the library into your application:

```javascript
import Amplify from 'aws-amplify';
import { AmplifyMapLibreRequest } from "maplibre-gl-js-amplify";
import awsconfig from './aws-exports';

> Amplify.configure(awsconfig);

Reviewer 2: Do we need `maplibre-gl` to be imported at this point of the guide?

Author: No, because `maplibre-gl-js-amplify` uses it under the hood and we don't need to import it directly.

> **Note:** Make sure that `maplibre-gl-js-amplify` version `1.0.5` or above is installed.

Import the library into your application:

```javascript
import { AmplifyMapLibreRequest } from "maplibre-gl-js-amplify";
```

> Next, create and render the [Map](https://maplibre.org/maplibre-gl-js-docs/api/map/) with the help of [AmplifyMapLibreRequest's](https://github.com/aws-amplify/maplibre-gl-js-amplify/blob/main/API.md#amplifymaplibrerequest).

Reviewer 2: ```suggestion
Next, create and render the [Map](https://maplibre.org/maplibre-gl-js-docs/api/map/) with the help of [AmplifyMapLibreRequest](https://github.com/aws-amplify/maplibre-gl-js-amplify/blob/main/API.md#amplifymaplibrerequest).
```
